site/docs/v1/tech/installation-guide/docker.adoc

```diff
@@ -208,3 +208,76 @@ If you want to build your own image starting with our base image, the following
 ```
 docker pull fusionauth/fusionauth-app
 ```
+
+== Kickstart
+
+Using Docker with Kickstart is a powerful combination. Using these technologies together lets you:
+
+* Configure and share development environments
+* Create replicable bug reports 
+* Spin up auth instances with a well known, versioned set of data for continuous integration and testing
+
+All the normal limitations of Kickstart apply (the Kickstart will not run if the database has already been set up with an API key, for example).
+
+It's easy to get started with Kickstart, but you'll need to tweak your Docker Compose files a bit. Before you being, you'll need a valid `kickstart.json` file. Check out the link:/docs/v1/tech/installation-guide/kickstart/[Kickstart documentation] for more information on writing one.
```

Comment:
We also have a repo for example kickstart files, not sure if we have many to choose from yet, we could link that here, or it is probably already linked on the main kickstart page, so maybe no reason to do that here. 
https://github.com/FusionAuth/fusionauth-example-kickstart

Reply:
Yes, let's leave it. It's on the kickstart page and the example apps page.